src/api/routers/participants/participantsRouter.ts

```diff
@@ -210,7 +202,7 @@ export function createParticipantsRouter() {
     }
   );
 
-  participantsRouter.put('/', createParticipant);
+  participantsRouter.put('/', handleCreateParticipant);
```

Comment:
Renamed because I split the handler into two functions, one of which is called `createParticipant`